src/components/input-field/input-field.tsx

```diff
@@ -454,15 +454,21 @@ export class InputField {
     };
 
     private renderHelperLine = () => {
+        const text: string = this.value || '';
+        const length = text.length;
+
         if (!this.maxlength && !this.hasHelperText()) {
             return;
         }
 
         return (
-            <div tabIndex={-1} class="mdc-text-field-helper-line">
-                {this.renderHelperText()}
-                {this.renderCharacterCounter()}
-            </div>
+            <limel-helper-line
+                helperTextId={helperTextId}
```

Comment:
I kept this as is, but I don't think the implementation on line 38 is correct actually 👇 
```tsx
const helperTextId = 'tf-helper-text';
```
We need to generate random IDs for each `<limel-input-field />` on the page. Otherwise all input filed components will have identical IDs this way, and the intended usage of the ID will fail 👇 
```tsx
 if (this.hasHelperText()) {
            properties['aria-controls'] = helperTextId;
            properties['aria-describedby'] = helperTextId;
        }
```

Reply:
More on this here: https://github.com/Lundalogik/lime-elements/issues/2255